pkg/detectors/rootly/rootly_test.go

```diff
@@ -0,0 +1,85 @@
+package rootly
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/engine/ahocorasick"
+)
+
+var (
+	validPattern   = "rootly_0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
```

Comment:
Do Rootly keys always start with `rootly_`? If so, we should update our regex pattern to accommodate this.